dts/bindings/adc/lltc,ltc2451.yaml

```
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 Brill Power Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Linear Technology LTC2451 ADC
+
+compatible: "lltc,ltc2451"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+  conversion-speed:
+    type: int
+    default: 0
+    enum:
+      - 0 # 60 Hz
+      - 1 # 30 Hz
+    description: Set conversion speed
```

Explicitly setting the conversion speed in Hz will make reading the Devicetree more clear. This will have to be converted to the register values in above function, of course.

```suggestion
  conversion-speed:
    type: int
    default: 60
    enum:
      - 60
      - 30
    description: Set conversion speed in Hz
```

---

Resolved